traits-stubs/traits-stubs/api.pyi

```diff
@@ -8,7 +8,16 @@
 #
```

Comment:
We gave up on trying to keep track of actual start years for the copyright dates, and just normalised those start dates, so that we're using the same copyright header everywhere. In effect, the copyright header refers to the entire package.

That said, `traits-stubs` _is_ a separate package, so maybe we should be using just `2020` throughout that package.

Reply:
Unrelated to this PR; let's open a separate issue for this.

Reply:
> That said, traits-stubs is a separate package, so maybe we should be using just 2020 throughout that package.

Yup. _That_ is why I think the header year should be 2020.

Reply:
Opened a PR: #1227.